tools/version-node.js

```diff
@@ -9,6 +9,12 @@ function dayNumber() {
   return dayOfYear;
 }
 
+function calculateMinorVersion() {
+  var baseYear = 2020; // VER_MINOR starts at 4 in 2020
```

Comment:
Where is this rule from? Probably just make minor version the lower 2 digits of year number which make it more visual friendly?

Reply:
It's documented here - https://github.com/microsoft/cpp_client_telemetry/blob/962db1ad555a0af634469f6ad7bb55b55c7c08a8/docs/versioning.md#release-and-version-strategy

VER_MINOR is 4. This number has to be incremented to 5 on January 1st, 2021.

Reply:
> Where is this rule from? Probably just make minor version the lower 2 digits of year number which make it more visual friendly?

Yes, can be done. Just prefer to maintain the incremental number from previous releases as of now.